src/modules/main/controllers/admin-write.controller.ts

```diff
@@ -98,6 +98,61 @@ export class AdminWriteController {
     return { id: artwork.id };
   }
 
+  @Post('artwork/nft/create')
+  @FormDataRequest()
+  async createArtworkFromNft(@Body() dto: CreateArtworkDto, @GetUserId() userId: UserId) {

```

Comment:
please create a separate dto for this endpoint as the original does not need nft and this one does not need image but nft is required